### ISSUE TITLE
Add resources for react

### ISSUE THREAD

**Issue by arpit456jain** [arpit456jain/Web-Development-Path-And-Resources#11]

**Comment by Sonakshi1901** [comment]
I would like to work on this issue

**Comment by arpit456jain** [comment]
Yes sure @Sonakshi1901

**Comment by SAHU-01** [comment]
@arpit456jain  do take a look at this react resource https://www.youtube.com/watch?v=QFaFIcGhPoM&list=RDCMUC80PWRj_ZU8Zu0HSMNVwKWw&start_radio=1&rv=QFaFIcGhPoM&t=1 and maybe Sonakshi can add it if you folks believe it can be helpful!

**Comment by arpit456jain** [comment]
Yes sure @SAHU-01 
Infact if you want you can add it too 🔥

**Comment by Sonakshi1901** [comment]
Will my name be added to the contributions list? @arpit456jain

**Comment by arpit456jain** [comment]
Yes for sure @Sonakshi1901 infact if you see the contributors section of GitHub you are already there

**Comment by arpit456jain** [comment]
Join discord server if you like
Link is in read me

**Comment by Sonakshi1901** [comment]
> Yes for sure @Sonakshi1901 infact if you see the contributors section of GitHub you are already there

I meant the one in the readme file.

**Comment by arpit456jain** [comment]
Yeah it will be updated soon
